src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3181,6 +3181,12 @@ protected BoundExpression BindObjectCreationExpression(ObjectCreationExpressionS
         {
             var type = BindType(node.Type, diagnostics);
 
+            if (node.Type.Kind() == SyntaxKind.TupleType)
+            {
+                Debug.Assert(node.HasErrors, "new <tuple type> should be a syntax error");
+                return BadExpression(node, LookupResultKind.NotCreatable);
```

Comment:
Considering that it might parse into something else in the future, I do not want to bind the type.